.github/workflows/publish.yml

```diff
@@ -32,6 +32,44 @@ jobs:
           name: engine
           path: target/wasm32-wasi/release/javy_core.wasm
 
+      - name: Archive quickjs_provider
+        run: gzip -k -f target/release/javy_quickjs_provider.wasm && mv target/release/javy_quickjs_provider.wasm.gz javy-quickjs_provider-${{ github.event.release.tag_name }}.wasm.gz
+
+      - name: Upload quickjs_provider to artifacts
+        uses: actions/upload-artifact@v3
+        with:
+          name: javy-quickjs_provider-${{ github.event.release.tag_name }}.wasm.gz
```

Comment:
Should this be `javy_quickjs_provider-{version}`? To be consistent with the generated wasm name. Or even `javy_quickjs_provider_{version}` since I don't see a reason to have a `-` there.